tools/message-screenshot.js

```diff
@@ -57,10 +57,15 @@ async function run() {
         await page.goto(`${options.realmUri}/#narrow/id/${options.messageId}`, {
             waitUntil: "networkidle2",
         });
-        const messageSelector = `#zfilt${CSS.escape(options.messageId)}`;
+        // TODO: Check if this works correctly.
```

Comment:
I tested this using `./tools/generate-integration-docs-screenshot` and it indeed works.

Reply:
Thanks, removed the TODO.